README.md

```diff
@@ -1,23 +1,67 @@
 # eris ![Logo][eris-logo]
+
 [![GoDoc][doc-img]][doc] [![Build][ci-img]][ci] [![Coverage Status][cov-img]][cov] [![GoReport][report-img]][report] [![Discord][chat-img]][chat] [![Mentioned in Awesome Go][awesome-img]][awesome]
 
-Package eris provides a better way to handle, trace, and log errors in Go. This package is inspired by a few existing packages: [xerrors](https://github.com/golang/xerrors), [pkg/errors](https://github.com/pkg/errors), and [Go 1.13 errors](https://golang.org/pkg/errors/).
+Package `eris` provides a better way to handle, trace, and log errors in Go.
 
 `go get github.com/rotisserie/eris`
 
-## How is eris different?
+## Why you'll want to switch to eris
+
+`eris` was inspired by a simple question: what if you could fix a bug without wasting time replicating the issue or digging through the code? With `eris`, you can achieve that goal by wrapping and outputting errors to your JSON logs:
+
+```json
+{
+  "error":{
+    "root":{
+      "message":"error internal server",
+      "stack":[
+        "eris_test.GetRelPath: /Users/roti/go/src/github.com/rotisserie/eris/logger_test.go: 81",
+        "eris_test.ProcessResource: /Users/roti/go/src/github.com/rotisserie/eris/logger_test.go: 98",
+        "eris_test.ProcessResource: /Users/roti/go/src/github.com/rotisserie/eris/logger_test.go: 102",
+        "eris_test.ExampleLogger: /Users/roti/go/src/github.com/rotisserie/eris/logger_test.go: 138",
+        "eris_test.TestExampleLogger: /Users/roti/go/src/github.com/rotisserie/eris/logger_test.go: 152"
```

Comment:
Example output needs update, also a suggestion to make it shorter if you can

Reply:
Also, i think we need to talk about example first and then put the output. A better approach would be to write a very simple 5 line code with eris and produce the JSON output and show it. Then we move onto more advanced use case showcasing the example you have, what do you think?